### PR TITLE
Completed es6-template-literals

### PR DESCRIPTION
**Pull request by kaitlin-lovrich**
image

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Javascript ES6 - Template Literals</title>
+  </head>
+  <body>
+    <script>
+      const number = 9200;
+      const street = 'Irvine Center Drive';
+      const address = number + ' ' + street;
```
Comment (shawnkost) [inline es6-template-literals/index.html]: This should be a template literal

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      const age = 25;
+
+      const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+      console.log('Bio:', bio);
+
+      const fruit = 'apple';
+      const item = 'pen';
+
+      const applePen = fruit + item;
```
Comment (shawnkost) [inline es6-template-literals/index.html]: This should be a template literal

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Kaitlin! We are only supposed to be converting the string concatenation to template literals. Nothing else

**Review by shawnkost — APPROVED** [review]
👍🏻

File: es6-template-literals/index.html (hunk 2 above)
Comment (kaitlin-lovrich) [reply es6-template-literals/index.html]: You got it. My apologies. I misunderstood the directions.

**Pull request merged**